email/provider_test.go

```diff
@@ -61,16 +64,19 @@ func testAccCheckEmailExists(n string) resource.TestCheckFunc {
 	}
 }
 func mockSendMailReturn421(addr string, a smtp.Auth, from string, to []string, msg []byte) error {
+	testRetries += 1
 	return errors.New("421 Service not available")
 }
 
 // function to test non-421 error
 func mockReturn500(addr string, a smtp.Auth, from string, to []string, msg []byte) error {
```

Comment:
```suggestion
func mockSendMailReturn500(addr string, a smtp.Auth, from string, to []string, msg []byte) error {
```

Didn't catch this before